packages/sku/config/eslint/importOrderConfig.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
         ],
         pathGroups: [
           {
-            pattern: '*.+(treat|less|css)',
+            pattern: '*.+(less|css)',
```

Comment:
```suggestion
            pattern: '*.@(less|css)',
```

https://github.com/micromatch/picomatch#extglobs